crates/turborepo-telemetry/src/events/generic.rs

```diff
@@ -212,6 +213,30 @@ impl GenericEventBuilder {
         self
     }
 
+    pub fn track_global_dot_env(&self, global_dot_env: Option<&[RelativeUnixPathBuf]>) -> &Self {
+        self.track(Event {
+            key: "global_dot_env".into(),
+            value: match global_dot_env {
+                Some(dot_env) => dot_env.len().to_string(),
+                None => "null".into(),
+            },
+            is_sensitive: EventType::NonSensitive,
+        });
+        self
+    }
+
+    pub fn track_dot_env(&self, dot_env: Option<&[RelativeUnixPathBuf]>) -> &Self {
+        self.track(Event {
+            key: "dot_env".into(),
```

Comment:
Currently we're producing one of these events per task. Not sure if we should roll these up to a single event or not.